plugins/render/render.go

```diff
@@ -0,0 +1,34 @@
+package render
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"os"
+)
+
+func ConfigPath() string {
+	configDir, err := os.UserConfigDir()
+	if err != nil {
+		return "~/.render/config.yaml"
+	}
+	return configDir + "/render/config.yaml"
+}
+
+func RenderCLI() schema.Executable {
+	return schema.Executable{
+		Name:    "Render CLI",
+		Runs:    []string{"render"},
+		DocsURL: sdk.URL("https://render.com/docs/cli"),
+		NeedsAuth: needsauth.IfAll(
+			needsauth.NotForHelpOrVersion(),
+			needsauth.NotWithoutArgs(),
```

Comment:
There are other commands which should not prompt for auth, such as `render config init`. Please do look if there are any others. 😄

Reply:
> There are other commands which should not prompt for auth, such as `render config init`. Please do look if there are any others. 😄

I have updated this.